project_tests.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -e
+
+proofreader src/
```

Comment:
Won't PHP-CS-Fixer not find the configuration as it's in the root? This also excludes the `test` directory etc.